pages/Colors.js

```diff
@@ -80,61 +75,27 @@ export default () => (
   <div
     className={`${colourStyles.container} ${colourStyles['column-container']}`}
   >
-    <h2>Color palette</h2>
-    <p>
-      Consistency of color is critical to maintaining a coherent family of
-      products, and accurately communicating the nature of specific content
-      within an application.
-    </p>
-    <p>
-      Colors are also used to distinguish where on the screen particular content
-      resides, such as on mobile devices where users need to be able to see
-      visual breaks while scrolling.
-    </p>
+    <section>
+      <h1>Colour Palette</h1>
+      <p>
+        Consistency of color is critical to maintaining a coherent family of
+        products, and accurately communicating the nature of specific content
+        within an application. Colours are also used to distinguish where on the
+        screen particular content resides, such as on mobile devices where users
+        need to be able to see visual breaks while scrolling.
+      </p>
+      <p>
+        For information about using colours to convey meaning, such as in
+        notifications or badges, read the{' '}
+        <a href="/colour-use" title="Using Colours">
```

Comment:
This should use a MenuItem component instead of a direct `<a>` 
`<MenuItem onClick={props.onClick} href="/colour-user" title="Feedback Colour Guide" />`

You can see an example in the `Navigation.js`. You'll also need to change `export default () => (` to `export default (props) => (`